Core/src/TrackFitting/MbfSmoother.cpp

```diff
@@ -22,7 +22,7 @@ void MbfSmoother::calculateSmoothed(InternalTrackState& ts,
 void MbfSmoother::visitNonMeasurement(const InternalTrackState& ts,
                                       BoundMatrix& big_lambda_hat,
                                       BoundVector& small_lambda_hat) const {
-  const auto F = ts.jacobian;
+  const InternalTrackState::Jacobian F = ts.jacobian;
```

Comment:
```suggestion
  const InternalTrackState::Jacobian & F = ts.jacobian;
```

Reply:
I think these are actually eigen map types so it is fine to copy them. but yeah I also always wonder if that might be an expensive assignment

Reply:
I think they are `EigenMaps`. Do we gain something from using references in this case?